doc/api/packages.md

```diff
@@ -0,0 +1,828 @@
+# Modules: Packages
+
+<!-- type=misc -->
+
+## Introduction
+
+A package is a folder described by a `package.json` file.
+
+A folder containing a `package.json` file, and all subfolders below that folder
+until the next folder containing another `package.json` file, are a _package
+scope_.
+
```

Comment:
I believe this sort of outline will be obvious from the generated table of contents since the fields are each their own section.

I'm not sure we should be listing fields that are not defined by Node.js. We do specifically link to the npm package.json definition page though in the package.json fields introduction.

Reply:
I've moved this to the beginning of the reference section itself in the feedback PR.